src/iface/interface/mod.rs

```diff
@@ -396,6 +396,14 @@ impl Interface {
     /// This function returns a boolean value indicating whether any packets were
     /// processed or emitted, and thus, whether the readiness of any socket might
     /// have changed.
+    ///
+    /// # Note
+    /// This function performs a bounded amount of work per call to avoid
+    /// starving other tasks of CPU time. If it returns true, there may still be
+    /// packets to be received or transmitted. Depending on system design,
+    /// calling this function in a loop may cause a denial of service: it is
+    /// safe if packets can be processed faster than they arrive, and unsafe
+    /// otherwise.
```

Comment:
Hmm, wouldn't "safe"/"unsafe" sound nebulously dangerous to many users? Maybe just `Depending on system design, calling this function in a loop may cause a denial of service (if packets cannot be processed faster than they arrive).`?

Reply:
That's a good point, since rust has a formal definition of `unsafe`

Reply:
Fixed in 2bcae4a6985d580a8bb6569b5a9d0e02f5be8602

Reply:
> Hmm, wouldn't "safe"/"unsafe" sound nebulously dangerous to many users?

I think a denial of service is plenty unsafe though. (Memory unsafety is one particular type of unsafety.)